apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.ts

```diff
@@ -1,10 +1,189 @@
 import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
+import {
+  loadSqlEquiv,
+  queryOptionalRow,
+  queryAsync,
+  queryOneRowAsync,
+} from '@prairielearn/postgres';
+import * as error from '@prairielearn/error';
+import { CourseInstance, Lti13CourseInstanceSchema } from '../../../lib/db-types';
+import { selectCoursesWithEditAccess } from '../../../models/course';
+import {
+  selectCourseInstanceById,
+  selectCourseInstancesWithStaffAccess,
+} from '../../../models/course-instances';
+import {
+  Lti13CourseNavigationInstructor,
+  //Lti13CourseNavigationNotReady,
```

Comment:
If this isn't used, please remove it from the `.html.ts` file and remove this comment.